main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gorilla/mux"
+)
+
+// Movie struct
+type Movie struct {
+	ID       int    `json:"ID"`
+	Title    string `json:"Title"`
+	Director string `json:"Director"`
+}
+
+// Movies list
+var Movies []Movie
+
+func getMovies(w http.ResponseWriter, r *http.Request) {
+	csvFile, err := os.Open("movies.csv")
+	if err != nil {
+		json.NewEncoder(w).Encode(err)
+	}
+	fmt.Println("Successfully Opened CSV file")
```

Comment:
These comments are okay to inform during runtime where in the code you are right now, and if there's has been any errors or successes, but it's ideal to use some logger.

Give it a look into `log`, `logrus` or `zerolog` (there are many others, these are the ones I'm kind of familiarized).